images/win/scripts/ImageHelpers/PathHelpers.ps1

```diff
@@ -1,68 +1,56 @@
-function Test-MachinePath{
+function Get-SystemVariable {
     [CmdletBinding()]
     param(
-        [string]$PathItem
+        [string]$SystemVariable
     )
-
-    $currentPath = Get-MachinePath
-
-    $pathItems = $currentPath.Split(';')
-
-    if($pathItems.Contains($PathItem))
-    {
-        return $true
-    }
-    else
-    {
-        return $false
-    }
+    return [System.Environment]::GetEnvironmentVariable($SystemVariable, "Machine")
 }
 
-function Set-MachinePath{
+function Set-SystemVariable {
     [CmdletBinding()]
     param(
-        [string]$NewPath
+        [string]$SystemVariable,
+        [string]$Value
     )
-    Set-ItemProperty -Path 'Registry::HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Session Manager\Environment' -Name Path -Value $NewPath
-    return $NewPath
+    [System.Environment]::SetEnvironmentVariable($SystemVariable, $Value, "Machine")
+    return Get-SystemVariable $SystemVariable
 }
 
-function Add-MachinePathItem
-{
+function Get-MachinePath {
     [CmdletBinding()]
```

Comment:
Let's remove [CmdletBinding(), param() and return

Reply:
Indeed. Updated.